packages/api/src/augment/rpc.ts

```diff
@@ -152,6 +152,10 @@ declare module '@polkadot/rpc-core/types.jsonrpc' {
        * Returns the chain ID used for transaction signing at the current best block. None is returned if not available.
        **/
       chainId: AugmentedRpc<() => Observable<U64>>;
+      /**
+       * Returns current client version.
+       **/
+      clientVersion: AugmentedRpc<() => Observable<Text>>;
```

Comment:
Had a slight bug here affecting the TS types, this is fixed in https://github.com/polkadot-js/api/pull/2994 so it will end up in the correct web3 section.

Reply:
ha ok cool. yes I was thinking it was weird net and web3 ended up in the eth section